go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/OneOfOne/xxhash v1.2.5 // indirect
```

Comment:
can you undo all the changes in this file?

Reply:
done